service/src/main/java/bio/terra/workspace/app/controller/WorkspaceApiController.java

```diff
@@ -485,7 +489,9 @@ public ResponseEntity<ApiCreateCloudContextResult> createCloudContext(
 
     CloudPlatform cloudPlatform = CloudPlatform.fromApiCloudPlatform(apiCloudPlatform);
     if (cloudPlatform == CloudPlatform.AWS) {
-      featureService.featureEnabledCheck(FeatureService.AWS_ENABLED, userRequest.getEmail());
+      featureService.featureEnabledCheck(
+          FeatureService.AWS_ENABLED,
+          samService.getUserEmailFromSamAndRethrowOnInterrupt(userRequest));
```

Comment:
do we have a situation when the petSA create a workspace?

Reply:
I checked the created_by_email field in devel and did not find any pet sa emails.

Reply:
then why do we need to query sam for user email?

Reply:
User Request may not have the mail address filled, if it only has a access token. email address is however needed for feature check